### DISCUSSION TITLE
No module named 'airflow.www'

### DISCUSSION THREAD

**Discussion by r-matsuzaka** [apache/airflow#20958] [General]
### Describe the issue with documentation

Hi.
I am noob of Airflow.

I encountered the error mentioned as title.
It happened suddenly.

I checked the [stack overflow](https://stackoverflow.com/questions/53583633/how-to-resolve-error-no-module-named-airflow-www-while-starting-airflow-web/56309707) but it does not help...

Do you have any idea for fixing?
I have no idea what is wrong with me.

```
brabra@brabra:~/brabra$ airflow webserver -p 8080
  ____________       _____________
 ____    |__( )_________  __/__  /________      __
____  /| |_  /__  ___/_  /_ __  /_  __ \_ | /| / /
___  ___ |  / _  /   _  __/ _  / / /_/ /_ |/ |/ /
 _/_/  |_/_/  /_/    /_/    /_/  \____/____/|__/
[2022-01-19 23:17:52,376] {dagbag.py:500} INFO - Filling up the DagBag from /dev/null
[2022-01-19 23:17:52,545] {manager.py:512} WARNING - Refused to delete permission view, assoc with role exists DAG Runs.can_create Admin
Running the Gunicorn Server with:
Workers: 4 sync
Host: 0.0.0.0:8080
Timeout: 120
Logfiles: - -
Access Logformat: 
=================================================================            

Error: No module named 'airflow.www'

[2022-01-19 23:18:40,291] {scheduler_job.py:1114} INFO - Resetting orphaned tasks for active dag runs
[2022-01-19 23:19:32,196] {scheduler_job.py:1114} INFO - Resetting orphaned tasks for active dag runs
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Reply by potiuk** [reply]
You likely have wrong installation. If you want to trry out airflow, follow precisely our quick start guides: https://airflow.apache.org/docs/apache-airflow/stable/start/index.html (but make sure you follow them rigorously if you want to avoid problems). 

Unless you provide a reproducible steps that lead to the installation problem you have, this is a discussion not a bug. And It's highly unlikely you will get any help if you do not provide reproducible-enough case.

**Reply by r-matsuzaka** [reply]
Thank you so much for your quick reply.
Yes, I agree with you.
Then I close this issue.
